app/services/AuditService.scala

```diff
@@ -96,11 +96,13 @@ class AuditService @Inject()(auditConnector: AuditConnector){
 
   def auditVariationSubmitted(internalId: String,
                               payload: JsValue,
-                              response: VariationResponse
+                              variationResponse: VariationResponse
                              )(implicit hc: HeaderCarrier): Unit = {
     val hasField = (field: String) =>
       payload.transform((JsPath \ field).json.pick).isSuccess
 
+    val trustTaxable = Json.toJson(payload) \ "details" \ "trust" \ "details" \ "trustTaxable"
```

Comment:
utils.Constants includes a few of these. Could possibly be simplified to `Json.toJson(payload) \ TRUST \ DETAILS \ TAXABLE` or something similar

Reply:
Updated